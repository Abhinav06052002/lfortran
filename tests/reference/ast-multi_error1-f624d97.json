{
    "basename": "ast-multi_error1-f624d97",
    "cmd": "lfortran --continue-compilation --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/multi_error1.f90",
    "infile_hash": "aae363b17603b231b2a906564a03365ce9a53952c754ea31187f352c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-multi_error1-f624d97.stdout",
    "stdout_hash": "96284a0d5bc016f307ab7acb06d38d4fb5c0ca994f7d1e5a322de322",
    "stderr": "ast-multi_error1-f624d97.stderr",
    "stderr_hash": "8e354c62a28f565b652876a84c5e566842065a5429ce01c56fea19e1",
    "returncode": 0
}