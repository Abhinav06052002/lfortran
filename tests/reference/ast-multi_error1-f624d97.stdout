(TranslationUnit
    [(Program
        multi_error1
        (TriviaNode
            [(EndOfLine)
            (EndOfLine)]
            []
        )
        []
        []
        [(Declaration
            (AttrType
                TypeInteger
                []
                ()
                ()
                None
            )
            []
            [(z
            []
            []
            ()
            ()
            None
            ())]
            ()
        )]
        [(Assignment
            0
            z
            1
            ()
        )
        (Print
            0
            ()
            [z]
            ()
        )]
        []
    )]
)
